src/VisualStudio/Core/Def/Implementation/DesignerAttribute/DesignerAttributeIncrementalAnalyzer.cs

```diff
@@ -79,11 +84,6 @@ public async Task AnalyzeDocumentAsync(Document document, SyntaxNode bodyOpt, In
                 return;
             }
 
-            if (await IsCpsProjectAsync(document.Project, cancellationToken).ConfigureAwait(false))
-            {
-                return;
-            }
-
             // use tree version so that things like compiler option changes are considered
```

Comment:
Comment seems off from code? Not new, but something seems fishy here.

Reply:
sure. will fix the comment.